lib/slack/real_time/client.rb

```diff
@@ -160,6 +160,10 @@ def to_s
         end
       end
 
+      def reconnected?
+        @reconnected ||= false
```

Comment:
This seems unrelated, but regardless you don't want to have a side-effect on calling a property method of assigning anything. So this could be `!! @reconnected` to coerce it into a boolean, for example.

Reply:
I think I might have left this by accident while looking at https://github.com/slack-ruby/slack-ruby-bot/pull/244 >_<